drivers/dma/dma_silabs_siwx917.c

```diff
@@ -68,7 +74,7 @@ static inline int siwx917_dma_data_width(uint32_t data_width)
 	}
 }
 
-static inline int siwx917_dma_burst_length(uint32_t blen)
+static inline int siwx917_burst_length(uint32_t blen)
```

Comment:
Looking at how this is used, this should return `bool` and be called `siwx917_burst_lengh_is_valid()`.

Oh, another thing I noticed with all of your short inline functions: drop the inline. Only add that if you've actually profiled the resulting binary and concluded that the compiler doesn't do an adequate job itself. Otherwise this is considered premature optimization.